dbhydra/src/tables.py

```diff
@@ -620,9 +620,10 @@ def __init__(self, db1, name, columns=None, types=None, id_column_name = "id"):
     #     return cls(db1, name, columns, types, id_column_name=id_column_name)
 
     def initialize_columns(self):
-        information_schema_table = Table(self.db1, 'INFORMATION_SCHEMA.COLUMNS')
-        query = f"SELECT COLUMN_NAME FROM INFORMATION_SCHEMA.COLUMNS WHERE TABLE_SCHEMA = '{self.db1.DB_DATABASE}' AND  TABLE_NAME  = '" + self.name + "';"
-        columns = information_schema_table.select(query)
+        """
+        TODO Dominik: Check for usecases of this method. Isn't it somewhat duplicated by `get_all_columns`?
+        """
```

Comment:
No, its not, one is something like GET request, another one is initializing columns in self.columns -> I might make one of them private